### ISSUE TITLE
Regression: Unique constraint name is ignored when adding a column

### ISSUE THREAD

**Issue by denarced** [liquibase/liquibase#1083]
**Description**
There's a regression on an old bug: [CORE-1313](https://liquibase.jira.com/browse/CORE-1313).

**To Reproduce**

1. Clone denarced/uniquetest (Spring Boot project)
1. Change database information in src/main/resources/application.properties
1. Create the Postgres database (on Linux CLI: createdb nameOfDb)
1. Build (mvn clean install) and run (java -jar target/*.jar) the project

The main table should have two unique columns (_name_ and _sigi_) and both of them should have custom unique constraint names (_UQ_name_ and _UQ_sigi_) but only the one that was created with the table (_UQ_name_) has it. The details of the table (using psql):

```
uniquetest=> \d t_main
                                  Table "public.t_main"
 Column |         Type          | Collation | Nullable |             Default              
--------+-----------------------+-----------+----------+----------------------------------
 id     | bigint                |           | not null | generated by default as identity
 name   | character varying(32) |           |          | 
 sigi   | character varying(32) |           |          | 
Indexes:
    "t_main_pkey" PRIMARY KEY, btree (id)
    "UQ_name" UNIQUE CONSTRAINT, btree (name)
    "t_main_sigi_key" UNIQUE CONSTRAINT, btree (sigi)
```

**Expected behavior**
Both columns should have the defined unique constraint names.

┆Issue is synchronized with this [Jira Bug](https://datical.atlassian.net/browse/LB-194) by [Unito](https://www.unito.io/learn-more)

**Comment by molivasdat** [comment]
@denarced Thanks for taking the time to write up this issue and Thanks for the detailed recreation scenario and making it so easy to reproduce.

**Comment by suilven** [comment]
@molivasdat I'm happy to work on this issue.

**Comment by cairngorm** [comment]
Unable to reproduce using Liquibase `3.10.2`.

**Comment by suilven** [comment]
Unable to reproduce using Liquibase `3.10.2`.

**Comment by molivasdat** [comment]
Hi @suilven thanks for checking into this.  @denarced Can you reproduce this with v3.10.2?  Also what version did your issue occur on?

**Comment by denarced** [comment]
Hi @molivasdat. I cannot reproduce the issue with version 3.10.2. Originally the issue occurred with version 3.8.8.

**Comment by molivasdat** [comment]
@denarced Thanks for validating this.
Closing issue as fixed in 3.10+
